### PR TITLE
Updated Scatter.rmd

### PR DESCRIPTION
**Pull request by nsshommo**
Updated Examples

### CODE REVIEW

File: website/scatter.Rmd [hunk 1]
```diff
@@ -13,13 +13,20 @@ knitr::opts_chunk$set(cache = TRUE)
...
 ```{r scatter-1}
 mpg
 ```
 
-
 ## RStudio's view pane
 
+View `mpg` in RStudio's dataset pane (via `View()`). Display all Audis sorted by year then cylinder. What is the maximum number of highway miles per gallon in this dataset?
+
+    ```
```
Comment (krlmlr) [inline website/scatter.Rmd]: Can you please unindent?

File: website/scatter.Rmd [hunk 2]
```diff
@@ -31,6 +38,15 @@ View(mpg)
 
 ## Scatterplots
 
+Create a few scatterplots using the following template (here with `hwy` versus `displ`):
+
+    ```{r echo=TRUE, eval=FALSE}
```
Comment (krlmlr) [inline website/scatter.Rmd]: Same here and below.

**Review by krlmlr — COMMENTED** [review]
Thanks, looks good. You can commit your updates to the same branch and push, the PR will be updated.

File: website/scatter3.Rmd [hunk 3]
```diff
@@ -114,3 +142,5 @@ ggplot(data = mpg) +
 ```
 
 Mapping the same variable to two aesthetics is sometimes useful to further emphasize this variable.
+
```
Comment (krlmlr) [inline website/scatter3.Rmd:146]: Can you please add captions before these pointers here and elsewhere?

File: website/scatter3.Rmd [hunk 4]
```diff
@@ -49,6 +56,15 @@ ggplot(data = mpg) +
 
 The `factor()` function helps:
 
+Experiment with the "color", "shape", "size", and "alpha" aesthetics.  Which combinations of attribute class (categorical/continuous) and aesthetics work well, which don't? Expand on the more surprising examples in the previous exercises.
```
Comment (krlmlr) [inline website/scatter3.Rmd]: All assignments belong right below the caption, this one after line 42.

File: website/position.Rmd [hunk 5]
```diff
@@ -97,3 +111,11 @@ ggplot(data = mpg) +
 ```
 
 Doesn't look useful.
+
```
Comment (krlmlr) [inline website/position.Rmd:114]: Can you please add a caption?

File: website/manual.Rmd [hunk 6]
```diff
@@ -69,13 +91,43 @@ ggplot(data = mpg) +
...
 
 - "shape": integers from 0 to 25, negative integers seem to display a letter
 
 - "size": positive numbers
+
+## More
+
+Find how to update the default point color for all scatterplots in the help for `update_geom_defaults()`. Does the setting persist between plots? Is it still active after restarting RStudio (Session → Restart R or Ctrl + Shift + F10, Cmd + Shift + F10 on the Mac).
+
+Find how to update the default point color for all label texts in the help for `theme()`. How to set or update the theme for all subsequent plots?
```
Comment (krlmlr) [inline website/manual.Rmd:129]: Please add one caption for each new exercise.

File: website/manual.Rmd [hunk 7]
```diff
@@ -69,13 +91,43 @@ ggplot(data = mpg) +
   )
 ```
 
+What happens if you assign a constant, e.g. `3`, to the "size" aesthetic *inside* the `aes()` call?
```
Comment (krlmlr) [inline website/manual.Rmd]: Move up, please.

File: website/geoms.Rmd [hunk 8]
```diff
@@ -256,11 +336,14 @@ ggplot(mpg) +
...
 
-
 ```{r geoms-23}
 ggplot(mpg) +
   geom_freqpoly(
     aes(hwy, color = drv),
     binwidth = 2
   )
 ```
+
```
Comment (krlmlr) [inline website/geoms.Rmd:346]: Please add captions.

File: website/geoms.Rmd [hunk 9]
```diff
@@ -40,6 +50,20 @@ ggplot(data = mpg) +
   geom_rug(mapping = aes(x = displ, y = hwy))
 ```
 
+What does `geom_rug()` do? Try to reduce overplotting with transparency or by adding `position = "jitter"`. How do you reduce overplotting for the points layer?
```
Comment (krlmlr) [inline website/geoms.Rmd:45]: Please move up.

File: website/facet.Rmd [hunk 10]
```diff
@@ -97,3 +116,10 @@ ggplot(data = mpg) +
     scales = "free"
   )
 ```
+
+## More
+
+Experiment with other arguments to `facet_wrap()` and `facet_grid()`.
+
+Find more exercises in Section 3.5.1 of r4ds.
```
Comment (krlmlr) [inline website/facet.Rmd]: One caption per exercise please.

**Review by krlmlr — COMMENTED** [review]
Thanks, looks good so far.

File: website/scatter3.Rmd (hunk 3 above)
Comment (nsshommo) [reply website/scatter3.Rmd:146]: What do you mean by captions?

Comment (krlmlr) [reply website/scatter3.Rmd:146]: captions -> subsections, starting with `##`

File: website/facet.Rmd (hunk 10 above)
Comment (krlmlr) [reply website/facet.Rmd]: "Experiment with ..." and "Find more ..." each need a `##` section.

File: website/geoms.Rmd [hunk 11]
```diff
@@ -256,11 +336,16 @@ ggplot(mpg) +
...
 ggplot(mpg) +
   geom_freqpoly(
     aes(hwy, color = drv),
     binwidth = 2
   )
 ```
+
+## More
+
+# Visualize the distribution of fuel economy in the dataset per drivetrain.  Do you prefer a histogram or a frequency polygon?
```
Comment (krlmlr) [inline website/geoms.Rmd]: We don't need `#` here, just a `##` section header below this sentence

File: website/manual.Rmd [hunk 12]
```diff
@@ -69,13 +100,34 @@ ggplot(data = mpg) +
...
 
 - "alpha": numbers between 0 and 1
 
 - "shape": integers from 0 to 25, negative integers seem to display a letter
 
 - "size": positive numbers
+
+## More
+
+# Find how to update the default point color for all scatterplots in the help for `update_geom_defaults()`. Does the setting persist between plots? Is it still active after restarting RStudio (Session → Restart R or Ctrl + Shift + F10, Cmd + Shift + F10 on the Mac).
```
Comment (krlmlr) [inline website/manual.Rmd]: Same as above.

File: website/geoms.Rmd (hunk 11 above)
Comment (nsshommo) [reply website/geoms.Rmd]: Done

File: website/manual.Rmd (hunk 12 above)
Comment (nsshommo) [reply website/manual.Rmd]: Done

**Pull request closed without merging**